app/views/layouts/application.html.slim

```diff
@@ -1,48 +1,50 @@
-            li.name
-              h1
-                =link_to 'Help with fees', root_url
-                span.phase-banner
-                  strong.phase-tag BETA
-            -unless hide_login_menu?(request.env['PATH_INFO'])
-              li.toggle-topbar.menu-icon
-                a href="#"
-                  span
-          -unless hide_login_menu?(request.env['PATH_INFO'])
-            - if user_signed_in?
-              section.top-bar-section
-                ul.right
-                  li
-                    span Welcome #{current_user.name}
-                  li= link_to "View profile", user_path(current_user.id)
-                  -if current_user.elevated?
-                    li= link_to 'View office', current_user.office
-                    li= link_to 'View staff', users_path
-                  li= link_to 'Sign out', destroy_user_session_path, :method => :delete
-      .row
-        .large-12.columns
-          /![if IE]><h1 class='ie-warning'>#{t('out_of_box.ie_warning')}</h1><![endif]
-          - flash.each do |key, value|
-            .alert-box class="#{key}" data-alert=''
-              |#{value.html_safe}
-              = link_to '&times;'.html_safe, '#', class: 'close'
-      .row
-        .large-12.columns
-          = yield
-    = render :partial => 'shared/footer'
+- content_for :page_title
+  = Rails.application.config.app_title
+
+- content_for :stylesheets
+  = stylesheet_link_tag('application.css')
+
+- content_for :head
+  = csrf_meta_tags
+  = yield :head_scripts
+
+- content_for :content_override
+  #wrapper.group
+    .container
+      .phase-banner
+        p
+          strong.phase-tag= Rails.application.config.phase.upcase
+          span
+            | Tell us what you think and #{link_to 'help us improve', Rails.application.config.feedback_url}
+    .container
+      section#content.cf
+        /![if IE]><h1 class='heading-large ie-warning'>#{t('out_of_box.ie_warning')}</h1><![endif]
+        - flash.each do |key, value|
+          .alert-box class="#{key}" data-alert=''
+            | #{value.html_safe}
+            = link_to '&times;'.html_safe, '#', class: 'close'
+
+        = yield
+
+- content_for :proposition_header
+  .header-proposition
+    .content
+      - if user_signed_in?
+        a.js-header-toggle.menu href="#proposition-links"  Menu
+      nav#proposition-menu
+        a#proposition-name href="/" = Rails.application.config.app_title
+        - if user_signed_in?
+          ul#proposition-links
+            li
+              span Welcome #{current_user.name}
+            li= link_to "View profile", user_path(current_user.id)
+            -if current_user.elevated?
+              li= link_to 'View office', current_user.office
+              li= link_to 'View staff', users_path
+            li= link_to 'Sign out', destroy_user_session_path, :method => :delete
+
+
+- content_for :javascripts
+  = javascript_include_tag('application.js')
+
+= render template: "layouts/local-template"
```

Comment:
Have you looked at using the layout from `layouts/moj_template`, which is part of the `moj_template`?